lib/labels/helpers.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package labels provides a way to get dynamic labels. Used by SSH, App,
+// and Kubernetes servers.
+package labels
+
+import (
+	"context"
+	"sync"
+
+	"github.com/gravitational/teleport/api/types"
+)
+
+var _ Importer = NewStaticImporter(nil)
```

Comment:
I'm a big fan of compile-time checks, though I could see an argument that running a constructor in a global might be bad.

Tweaked this to be inside an anonymous function to avoid any actual work being done.